lib/redis-client.js

```diff
@@ -40,6 +40,34 @@ RedisClient.prototype.getLink = function(link) {
   })
 }
 
+RedisClient.prototype.getLinks = function() {
+  let links = []
+  const scan = (start) => {
+    return new Promise((resolve, reject) => {
+      this.client.scan(start, 'match', '/*', (err, results) => {
+        if (err) {
+          return reject(err)
+        }
+        return resolve(results)
+      })
+    }).then(([cursor, results]) => {
+      links.push(...results)
+      if (parseInt(cursor) !== 0) {
+        return scan(cursor)
+      } else {
+        return links
+      }
+    })
+  }
+
+  return scan(0).then((links) => Promise.all(links.map(x =>
+      this.getLink(x).then(linkData => {
```

Comment:
Two things:

* For a future revision, I think we'll want to update this to use the [MGET][] command for efficiency's sake. (If you want to take a stab at it now, though, feel free!)
* Update `x` to `link` just for readability's sake.

[MGET]: https://redis.io/commands/mget